mmv1/third_party/terraform/tests/resource_dataflow_flex_template_job_test.go.erb

```diff
@@ -104,91 +116,335 @@ func TestAccDataflowFlexTemplateJob_streamUpdateFail(t *testing.T) {
 	})
 }
 
+func TestAccDataflowFlexTemplateJob_FullUpdate(t *testing.T) {
```

Comment:
These tests looks great! Do you know if it is feasible to add one more test that uses a batch job and tries to update a field like `machine_type`, to test that the customize diff is working as intended?

Reply:
hi @roaks3,

I am unable to get a working test for the batch update, the api call to update the jobs is made the launcher machine from a new jobID. so the error message is not returned to the client. we have to check for the logging api.